### ISSUE TITLE
slash / backslash in partname

### ISSUE THREAD

**Issue by sp-apertus** [openpnp/openpnp#677]
# Problem Report
## Describe the Issue
When the board contains a part with a slash or backslash in the name the template creation will fail, namely WritePartTemplateImage will not report an error but simple not create any template images.

**Comment by vonnieda** [comment]
Makes sense, should be a simple fix to fix up the incoming name. We should probably just remove all punctuation aside from period, dash and underscore.
